### ISSUE TITLE
Any plan to release the checkpoint?

### ISSUE THREAD

**Issue by windygoo** [cvlab-stonybrook/SAMPath#5]
My reproduced results are {'Dice': tensor(0.7739, device='cuda:0'), 'Dice_Bal': tensor(0.6828, device='cuda:0'),  'F1': tensor(0.7739, device='cuda:0'), 'IOU_Jaccard': tensor(0.6311, device='cuda:0'), 'IOU_Jaccard_Bal': tensor(0.5344, device='cuda:0')}, which are  lower than the reported results by a significant margin. 

Dice: 0.7739 (reproduced) vs 0.7949 (in paper)
IoU: 0.6311 (reproduced) vs 0.6596 (in paper)

Is any plan to release the checkpoint?

**Comment by jingweizhang-xyz** [comment]
The training log of this can be found at: https://wandb.ai/jingwezhang/sam_finetune_loss/reports/BCSS_fusion_focal_0125_iou_00625--Vmlldzo2MzMyMTk3
The model can be found at the end of this page.

**Comment by windygoo** [comment]
> The training log of this can be found at: https://wandb.ai/jingwezhang/sam_finetune_loss/reports/BCSS_fusion_focal_0125_iou_00625--Vmlldzo2MzMyMTk3 The model can be found at the end of this page.

![image](https://github.com/cvlab-stonybrook/SAMPath/assets/109289312/64df4d88-e9c7-4893-acaf-9cdeeacf1a65)

**Comment by jingweizhang-xyz** [comment]
Sorry, wrong link. https://api.wandb.ai/links/jingwezhang/2mov7snx

**Comment by windygoo** [comment]
> Sorry, wrong link. https://api.wandb.ai/links/jingwezhang/2mov7snx

Get it, Thanks!

**Comment by windygoo** [comment]
> Sorry, wrong link. https://api.wandb.ai/links/jingwezhang/2mov7snx

The checkpoint at the last epoch is saved? not the best on the validation set?

**Comment by ryankish** [comment]
I'm wondering if you might also be able to share the Fine-tuned SAM (w.o. $F_p$) BCSS checkpoint? Thank you.
